lib/slax/commands/github_commands.ex

```diff
@@ -138,7 +151,7 @@ defmodule Slax.Commands.GithubCommands do
     end)
     |> Enum.split_with(fn
       {:ok, _, _} -> true
-      {:error, _} -> false
+      {:error, _, _} -> false
```

Comment:
the previous `Enum.map` was generating either 2-element `:ok` tuples or 3-element `:error` tuples but `split_with` was matching on a 2-element `:error` tuple here. This resulted in `split_with` raising an error any time there was an error creating an issue.